runtime/syntax/rsc.vim

```diff
@@ -0,0 +1,82 @@
+" Vim syntax file
+" Language:        RouterOS scripts
+" Maintainer:      zainin
+" Original Author: ndbjorne @ MikroTik forums
+" Last Change:     2017-07-26
+
+" quit when a syntax file was already loaded
+if exists("b:current_syntax")
+  finish
+endif
+
+syn case ignore
+
+set iskeyword=A-Z,a-z
```

Comment:
this needs to be `:syn iskeyword`

Reply:
Turns out it wasn't really necessary, so I've removed this line.

Reply:
There's a few hyphenated keywords like `as-value` that are being missed because `'iskeyword'` doesn't include `-`.  If you don't want to use `syn-iskeyword` you could make them a `syn-match`.

Reply:
Sorry, I didn't notice `-` isn't included by default. It should be better now.